ReactAndroid/src/main/java/com/facebook/react/animated/NativeAnimatedNodesManager.java

```diff
@@ -38,16 +45,18 @@
  *
  * IMPORTANT: This class should be accessed only from the UI Thread
  */
-/*package*/ class NativeAnimatedNodesManager {
+/*package*/ class NativeAnimatedNodesManager implements EventDispatcherListener {
 
   private final SparseArray<AnimatedNode> mAnimatedNodes = new SparseArray<>();
   private final ArrayList<AnimationDriver> mActiveAnimations = new ArrayList<>();
   private final ArrayList<AnimatedNode> mUpdatedNodes = new ArrayList<>();
+  private final Map<String, EventAnimationDriver> mEventDrivers = new HashMap<>();
   private final UIImplementation mUIImplementation;
   private int mAnimatedGraphBFSColor = 0;
 
-  public NativeAnimatedNodesManager(UIImplementation uiImplementation) {
+  public NativeAnimatedNodesManager(UIImplementation uiImplementation, EventDispatcher eventDispatcher) {
     mUIImplementation = uiImplementation;
+    eventDispatcher.addListener(this);
```

Comment:
I think we should still send events to JS since some component could do some JS logic in their callback and there is also the event listener that we would still need to call into js for https://github.com/facebook/react-native/blob/master/Libraries/Animated/src/AnimatedImplementation.js#L1887